### DISCUSSION TITLE
c3270 core code upstreamed with TLS and localhost now functioning

### DISCUSSION THREAD

**Discussion by MikeFultonDev** [zopencommunity/meta#827] [Announcements]
The latest version of c3270 now has full TLS support for encrypted connections and now properly supports 'localhost' as a specification. The fix for this problem was in the core code (timing issue) and is fixed in the latest drop. As well, patches we needed for z/OS have been upstreamed. The patches we have now are for the test suites, not the core utilities.

There are a few known issues that I believe are minor: https://github.com/ZOSOpenTools/c3270port/issues 
If you feel otherwise - i.e. one of the issues prevents you from using c3270 - please either open an issue or comment on an already existing issue.

If there are things you find lacking with c3270, that would be also useful to open an issue for - as an 'enhancement'.

**Comment by v1gnesh** [comment]
> in the latest drop

![image](https://github.com/ZOSOpenTools/meta/assets/234149/60bb1316-248b-4526-9ee3-5bf637812002)
